src/yggdrasil/core_test.go

```diff
@@ -1,3 +1,5 @@
+// +build debug
```

Comment:
I'm not sure I like disabling this outside of debug builds, since it shouldn't be built outside of `go test` anyway, and hiding tests outside of debug builds may just lead to hiding errors (or problems with the tests)